src/redis_lock/__init__.py

```diff
@@ -197,11 +231,13 @@ def __exit__(self, exc_type=None, exc_value=None, traceback=None, force=False):
         if self._lock_renewal_thread is not None:
             self._stop_lock_renewer()
         logger.debug("Releasing %r.", self._name)
-        try:
-            self._client.evalsha(UNLOCK_SCRIPT_HASH, 2, self._name, self._signal, self._id)
-        except NoScriptError:
+        is_cached, _ = _redis_eval(
```

Comment:
It would show the warning for all the scripts. That what I was actually asking for :)

My intention with the warning was to have a clear way to see if it's cached or not (eg: for tests, or to see if they are getting flushed out - like a too busy redis server).